Assignment/Program.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter input number :");
+   if (ulong.TryParse (Console.ReadLine (), out ulong n)) {
+      ulong firstTerm = 0, secondTerm = 1;
+      for (ulong i = 0; i < n; i++) {
+         Console.WriteLine ($"f{i} : {firstTerm}");
+         ulong thirdTerm = firstTerm + secondTerm;
+         (firstTerm, secondTerm) = (secondTerm, thirdTerm);
       }
-   }
+      break;

```

Comment:
Why do you need this break?

Reply:
Once the Fibonacci series is done for user input the loop should end